src/lsp/cobol_cfg/cfg_options.ml

```diff
@@ -0,0 +1,43 @@
+(******************************************************************************)
+(*                                                                            *)
+(*     Copyright (c) 2021-2023 OCamlPro SAS                                   *)
+(*                                                                            *)
+(*     All rights reserved.                                                   *)
+(*     This file is distributed under the terms of the                        *)
+(*     OCAMLPRO-NON-COMMERCIAL license.                                       *)
+(*                                                                            *)
+(******************************************************************************)
+
+type transformation =
+  | Descendents of int
+  | Neighborhood of int
+
+type t = {
+  graph_name: string option;
+  hide_unreachable: bool;
+  collapse_fallthru: bool;
+  shatter_hubs: int option;
+  transformation: transformation option;
+  hidden_nodes: int list;
+  split_nodes: int list;
+}
+
+let create
```

Comment:
We don't use this kind of creation function in these libraries. Directly creating the record should be straightforward enough.